odoo/addons/base/migrations/11.0.1.3/pre-migration.py

```diff
@@ -25,6 +25,22 @@
 ]
 
 
+def handle_partner_sector(env):
+    if openupgrade.table_exists(env.cr, 'res_partner_sector'):
+        # Module `partner_sector` was installed in 10.0
+        openupgrade.rename_models(
+            env.cr, [('res.partner.sector', 'res.partner.industry')],
+        )
+        openupgrade.rename_tables(
+            env.cr, [(('res_partner_sector', 'res_partner_industry'))]
+        )
+        openupgrade.rename_fields(
+            env.cr, [
```

Comment:
should be `env` instead of `env.cr`

Reply:
Totally! fixed

Reply:
but the problem I see is that in the base pre-migration, the `env` still doesn't have the base models if remember correctly. 🤔

Reply:
Uhm, good question... let me check if we need it for something specific.

Reply:
There's no problem:

https://github.com/OCA/openupgradelib/blob/12b1ed9c7d2c354499ba5f89067f908f0f8f94ad/openupgradelib/openupgrade.py#L413-L415

Reply:
good